.github/workflows/testing.yml

```diff
@@ -9,16 +9,16 @@ jobs:
     name: Drupal ${{ matrix.drupal-core }} (PHP ${{ matrix.php-versions }})
     runs-on: ubuntu-20.04
     env:
-      extensions: mbstring, xml, pdo_sqlite, gd, opcache
+      extensions: :sodium, Imagick, gd, PDO, mysql, pdo_mysql, mbstring, xmlwriter, pdo, date, dom, filter, hash, json, pcre, session, SimpleXML, SPL, tokenizer, xml, curl
```

Comment:
any reason why we need all those new extensions? We don't even test on mysql for example?

Reply:
shivammathur/setup-php has for PHP8.1 much less extensions enabled than on <=PHP8. This is to be safe that all extensions that drupal requires are there. We had the same issue in our Thunder distribution and @alexpott figured that out.

Reply:
ok strange, because our Drupal 9 tests with PHP 8.1 already work normally. Does Drupal 10 require new extensions? Or could we undo this line and it would work?